src/Model/Project.hs

```diff
@@ -146,6 +148,9 @@ ticketToOrderable (TaggedTicket ((Entity _ ticket), is_claimed, tags)) =
 
     search_literal str = (not . null . T.breakOnAll str) (ticketName ticket)
 
+tags :: TaggedTicket -> [AnnotatedTag]
+tags (TaggedTicket (Entity _ ticket, is_claimed, tags)) = tags
```

Comment:
This function has a name that is a bad combination of small, generic, and global/top-level, considering how specific its behavior is. Probably better to inline in wherever it is used, or give it a much more specific name.

For that matter, TaggedTicket is a bit too complex to be a newtype. A 4-tuple is a bit much. With record syntax you could automatically declare a function taggedTicketTags :: TaggedTicket -> [AnnotatedTag].